tools/livecd-iso-to-disk.sh

```diff
@@ -516,8 +516,7 @@ resetMBR() {
         fi
     fi
     # Wait for changes to show up/settle down.
-    udevadm settle
-    sleep 5
+    udevadm settle -E $TGTDEV
```

Comment:
It doesn't look like you're testing if `-E`/`--exit-if-exists` is a valid option?

Reply:
Wait, nevermind...